package.json

```diff
@@ -205,6 +206,7 @@
   "license": "GPL-3.0-or-later",
   "dependencies": {
     "@amplitude/analytics-browser": "^1.1.4",
+    "@cowprotocol/ethflowcontract": "git+https://github.com/cowprotocol/ethflowcontract.git#a06e19c",
```

Comment:
Good for now. Once we have the official releases, we should depend on release tags

Reply:
What about creating release candidates in the mean time?
Will look much cleaner